### ISSUE TITLE
Subspace example is broken in the documentation

### ISSUE THREAD

**Issue by aTrotier** [MagneticResonanceImaging/MRIReco.jl#190]
https://magneticresonanceimaging.github.io/MRIReco.jl/latest/generated/examples/03-subspaceReconstruction/

**Comment by aTrotier** [comment]
My bad, I thought a test was performed... I don't know what PR broke that :/

Error : 
```julia
ERROR: LinearOperatorException("shape mismatch")
Stacktrace:
  [1] mul!(res::Vector{…}, op::DiagOp{…}, v::Vector{…}, α::ComplexF32, β::ComplexF32)
    @ LinearOperators ~/.julia/packages/LinearOperators/ixrUQ/src/operations.jl:25
  [2] mul!
    @ ~/.julia/packages/LinearOperators/ixrUQ/src/operations.jl:40 [inlined]
  [3] *
    @ ~/.julia/packages/LinearOperators/ixrUQ/src/operations.jl:47 [inlined]
  [4] prox!(reg::TransformedRegularization{…}, x::Vector{…}, args::Float64)
    @ RegularizedLeastSquares ~/.julia/packages/RegularizedLeastSquares/LvIWc/src/Regularization/TransformedRegularization.jl:29
  [5] iterate(solver::ADMM{…}, state::RegularizedLeastSquares.ADMMState{…})
    @ RegularizedLeastSquares ~/.julia/packages/RegularizedLeastSquares/LvIWc/src/ADMM.jl:237
  [6] iterate(solver::ADMM{…})
    @ RegularizedLeastSquares ~/.julia/packages/RegularizedLeastSquares/LvIWc/src/RegularizedLeastSquares.jl:190
  [7] iterate
    @ ./iterators.jl:206 [inlined]
  [8] iterate
    @ ./iterators.jl:205 [inlined]
  [9] solve!(solver::ADMM{…}, b::Vector{…}; callbacks::Function, kwargs::@Kwargs{})
    @ RegularizedLeastSquares ~/.julia/packages/RegularizedLeastSquares/LvIWc/src/RegularizedLeastSquares.jl:111
 [10] solve!(solver::ADMM{…}, b::Vector{…})
    @ RegularizedLeastSquares ~/.julia/packages/RegularizedLeastSquares/LvIWc/src/RegularizedLeastSquares.jl:102
 [11] macro expansion
    @ ~/Documents/JULIA_PROJECT/MRIReco_fix_subspace/MRIReco.jl/src/Reconstruction/IterativeReconstruction.jl:396 [inlined]
 [12] ##reducing_function#244#32
    @ ~/.julia/packages/FLoops/GN1Bm/src/reduce.jl:817 [inlined]
 [13] (::InitialValues.AdjoinIdentity{MRIReco.var"##reducing_function#244#32"{…}})(x::Tuple{}, y::Tuple{Int64, Int64})
    @ InitialValues ~/.julia/packages/InitialValues/OWP8V/src/InitialValues.jl:306
 [14] next
    @ ~/.julia/packages/Transducers/fnznF/src/combinators.jl:290 [inlined]
 [15] next
    @ ~/.julia/packages/Transducers/fnznF/src/core.jl:287 [inlined]
 [16] macro expansion
    @ ~/.julia/packages/Transducers/fnznF/src/core.jl:181 [inlined]
 [17] macro expansion
    @ ~/.julia/packages/Transducers/fnznF/src/processes.jl:323 [inlined]
 [18] macro expansion
    @ ~/.julia/packages/Transducers/fnznF/src/simd.jl:41 [inlined]
 [19] _foldl_product
    @ ~/.julia/packages/Transducers/fnznF/src/processes.jl:322 [inlined]
 [20] _foldl_product
    @ ~/.julia/packages/Transducers/fnznF/src/processes.jl:312 [inlined]
 [21] __foldl__
    @ ~/.julia/packages/Transducers/fnznF/src/processes.jl:301 [inlined]
 [22] foldl_basecase
    @ ~/.julia/packages/Transducers/fnznF/src/processes.jl:361 [inlined]
 [23] _reduce_basecase(rf::Transducers.BottomRF{…}, init::Transducers.InitOf{…}, reducible::Transducers.SizedReducible{…})
    @ Transducers ~/.julia/packages/Transducers/fnznF/src/threading_utils.jl:58
 [24] _reduce(ctx::Transducers.NoopDACContext, rf::Transducers.BottomRF{…}, init::Transducers.InitOf{…}, reducible::Transducers.SizedReducible{…})
    @ Transducers ~/.julia/packages/Transducers/fnznF/src/reduce.jl:139
 [25] _transduce_assoc_nocomplete
    @ ~/.julia/packages/Transducers/fnznF/src/reduce.jl:131 [inlined]
 [26] transduce_assoc(xform::Transducers.IdentityTransducer, step::Transducers.AdHocRF{…}, init::Transducers.InitOf{…}, coll0::Base.Iterators.ProductIterator{…}; simd::Val{…}, basesize::Nothing, stoppable::Nothing, nestlevel::Nothing)
    @ Transducers ~/.julia/packages/Transducers/fnznF/src/reduce.jl:108
 [27] transduce(xf::Transducers.IdentityTransducer, rf::Function, init::Transducers.InitOf{…}, coll::Base.Iterators.ProductIterator{…}, exc::Transducers.ThreadedEx{…})
    @ Transducers ~/.julia/packages/Transducers/fnznF/src/executors.jl:152
 [28] transduce(xf::Transducers.IdentityTransducer, rf::Function, init::Transducers.InitOf{…}, coll::Base.Iterators.ProductIterator{…}, exc::Transducers.PreferParallel{…})
    @ Transducers ~/.julia/packages/Transducers/fnznF/src/executors.jl:164
 [29] _fold
    @ ~/.julia/packages/FLoops/GN1Bm/src/reduce.jl:851 [inlined]
 [30] _fold
    @ ~/.julia/packages/FLoops/GN1Bm/src/reduce.jl:849 [inlined]
 [31] macro expansion
    @ ~/.julia/packages/FLoops/GN1Bm/src/reduce.jl:829 [inlined]
 [32] reconstruction_multiCoilMultiEcho_subspace(acqData::AcquisitionData{…}; reconSize::Tuple{…}, reg::Vector{…}, sparseTrafo::Vector{…}, weights::Vector{…}, solver::Type{…}, senseMaps::Array{…}, encodingOps::Nothing, arrayType::Type{…}, params::@Kwargs{…})
    @ MRIReco ~/Documents/JULIA_PROJECT/MRIReco_fix_subspace/MRIReco.jl/src/Reconstruction/IterativeReconstruction.jl:381
 [33] reconstruction(acqData::AcquisitionData{Float32, 2}, recoParams::Dict{Symbol, Any})
    @ MRIReco ~/Documents/JULIA_PROJECT/MRIReco_fix_subspace/MRIReco.jl/src/Reconstruction/Reconstruction.jl:53
 [34] top-level scope
    @ ~/Documents/JULIA_PROJECT/MRIReco_fix_subspace/MRIReco.jl/docs/dev.jl:63
Some type information was truncated. Use `show(err)` to see complete types.
```

**Comment by aTrotier** [comment]
should be fixed by PR #207
